File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int>prefSum;
        int n=nums.size();
       int  sum=0;

        for(int i=0;i<n;++i){
         sum+=nums[i];
         prefSum.push_back(sum);
        }
        return prefSum;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int n=nums.length;
        int []arr=new int [n];
        int presum=0;

        for(int i=0;i<n;++i){
            presum+=nums[i];
            arr[i]=presum;
        }
        return arr;
    }
}